### PR TITLE
Updating Data Sources pages

### PR DESCRIPTION
**Pull request by ajaits**

### CODE REVIEW

File: datasets/index.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
 ---
 layout: default
 title: Data Sources
-nav_order: 110
```
Comment (shifucun) [inline datasets/index.md:4]: Can you revert this one? Or is it auto-generated?

**Review by pradh — APPROVED** [review]

File: datasets/index.md (hunk 1 above)
Comment (ajaits) [reply datasets/index.md:4]: This was autogenerated. Should i revert?

Comment (shifucun) [reply datasets/index.md:4]: yes, please. I should check why it changes this file...

Comment (ajaits) [reply datasets/index.md]: reverted this file

**Review by shifucun — APPROVED** [review]

**Review by n-h-diaz — APPROVED** [review]

**Pull request merged**